docs/databases/swift-kuery.html

```diff
@@ -21,37 +21,44 @@
-        <li><a onclick="localStorage.setItem('package', 'Swift-Kuery'); setActiveSidebarElementForParent('swift-kuery')" href="../getting-started/update-package.html#">Update Dependencies:</a> Swift Kuery needs adding to the Package.swift file.</li>
+          <li><a onclick="setActiveSidebarElementForParent('create-server')" href="../getting-started/create-server.html#">Kitura Server:</a> Learn how to create one in our Getting Started guide.</li>
+          <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Kitura Style Guide:</a> Follow our guide to see how a production ready server should be structured.</li>
+          <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Book Model:</a> You have defined a `Book` struct as described in Step 5 of the style guide.</li>
+          <li><a onclick="setActiveSidebarElementForParent('logging')" href="../getting-started/logging.html#">Logging:</a> Learn how to add logging to your project.</li>
+          <li><a onclick="localStorage.setItem('package', 'Swift-Kuery'); setActiveSidebarElementForParent('swift-kuery')" href="../getting-started/update-package.html#">Update Dependencies:</a> Your Swift Kuery plugin needs to be added to the Package.swift file.</li>
       </ul>
       <div class="underline"></div>
-      <h2 class="heading-2"><span class="blue-text">Step 1:</span> Create an `KueryRoutes.swift` file</h2>
+      <h2 class="heading-2"><span class="blue-text">Step 1:</span> Create the Kuery routes</h2>
       <p>Open your `Sources` > `Application` > `Application.swift`</p>
       <p> Inside the `postInit()` function add: </p>
       <pre><code class="language-swift">initializeKueryRoutes(app: self)</code></pre>
       <p>Create a new file called `KueryRoutes.swift` in `Sources` > `Application` > `Routes`</p>
       <p>Inside this file, add the framework for our routes code:</p>
-      <pre><code class="language-swift">import LoggerAPI
-  // Create connection pool and table here
+      <pre><code class="language-swift">import KituraContracts
+import LoggerAPI
 
-  func initializeKueryRoutes(app: App) {
+func initializeKueryRoutes(app: App) {
+    app.router.post("/kuery", handler: app.insertHandler)
+    app.router.get("/kuery", handler: app.selectHandler)
+}
 
-      app.router.post("/kuery") { request, response, next in
-          // Handle POST here
-          next()
-      }
-      app.router.get("/kuery") { request, response, next in
-          // Handle GET here
-          next()
-      }
-  }
-  // Write functions here
-  </code></pre>
+extension App {
+    // Create connection pool and initialize BookTable here
+
+    func insertHandler(book: Book, completion: @escaping (Book?, RequestError?) -> Void) {
+        // Handle POST here
+    }
+
+    func selectHandler(completion: @escaping ([Book]?, RequestError?) -> Void) {
+        // Handle GET here
+    }
+}</code></pre>
```

Comment:
(note not on this line) Create a Postgres Database -> Create a Postgres database